File: BankingTool.Model/Dto/RequestDto/BankAccount/IsCustomerHasCreditCardInThatBankRequest.cs
using BankingTool.Model.Dto.RequestDtos;

namespace BankingTool.Model.Dto.RequestDto.BankAccount
{
    public class IsCustomerHasCreditCardInThatBankRequestObject
    {
        public IsCustomerHasCreditCardInThatBankRequest Request { get; set; }
    }
    public class IsCustomerHasCreditCardInThatBankRequest
    {
        public IsCustomerHasCreditCardInThatBankRequestCustomer Customer { get; set; }
        public IsCustomerHasCreditCardInThatBankRequestBank Bank { get; set; }
    }
    public class IsCustomerHasCreditCardInThatBankRequestCustomer : RequestId
    {
    }
    public class IsCustomerHasCreditCardInThatBankRequestBank : RequestId
    {
    }
}

File: bankingtool.client/src/app/dto/request/bank-account/is-customer-has-credit-card-in-that-bank-request.ts
import { RequestId } from "../request-id";

export interface IsCustomerHasCreditCardInThatBankRequestObject {
  request: IsCustomerHasCreditCardInThatBankRequest;
}

export interface IsCustomerHasCreditCardInThatBankRequest {
  customer: IsCustomerHasCreditCardInThatBankRequestCustomer;
  bank: IsCustomerHasCreditCardInThatBankRequestBank;
}

export interface IsCustomerHasCreditCardInThatBankRequestCustomer extends RequestId {

}

export interface IsCustomerHasCreditCardInThatBankRequestBank extends RequestId {

}
